Makefile

```diff
@@ -0,0 +1,17 @@
+# This Makefile is used to compile an LND binary for the integration tests.
+
+GO_BUILD := go build
+CARGO_TEST := cargo test
+LND_PKG := github.com/lightningnetwork/lnd
+
+TMP_DIR := "/tmp"
+UNAME_S := $(shell uname -s)
+ifeq ($(UNAME_S),Darwin)
+        TMP_DIR=${TMPDIR}
+endif
+
+itest:
```

Comment:
First time run needs to pull the submodule code: 

`git submodule update --init --recursive`